cannon/mipsevm/tests/evm_multithreaded_test.go

```diff
@@ -1193,6 +1193,32 @@ func TestEVM_UnsupportedSyscall(t *testing.T) {
 	}
 }
 
+func TestEVM_EmptyThreadStacks(t *testing.T) {
+	t.Parallel()
+	var tracer *tracing.Hooks
+
+	cases := []struct {
+		name            string
+		activeStackSize int
+		otherStackSize  int
+		traverseRight   bool
+	}{
+		{name: "Empty stacks, traverse right", activeStackSize: 0, otherStackSize: 0, traverseRight: true},
+		{name: "Empty stacks, traverse left", activeStackSize: 0, otherStackSize: 0, traverseRight: false},
```

Comment:
if otherStackSize is non-zero it will across the target error line 166 to [172](https://github.com/ethereum-optimism/optimism/blob/5f625e0245d8f3fec4668239c2086308caf93d8e/packages/contracts-bedrock/src/cannon/MIPS2.sol#L172).  So should we have these otherStackSize is non-zero case?

> We should also go ahead and add some test cases where the `otherStackSize` is non-zero. I don't see any other tests that hit that case.